managed/services/management/dbaas/kubernetes_server.go

```diff
@@ -271,22 +270,23 @@ func replaceAWSAuthIfPresent(kubeconfig string, keyID, key string) (string, erro
 	return string(c), err
 }
 
-func installOLMOperator(ctx context.Context, client dbaasClient, kubeconfig, version string) error {
```

Comment:
@cfsalguero  it's unused code, do we need it?

Reply:
If unused, let's remove - we can add it back when we actually need it.